index.php

```diff
@@ -7,27 +7,34 @@
 <main id="main-content">
 
   <!-- main posts loop -->
-  <section id="posts">
+  <section id="posts" class="container">
 
 <?php
-if( have_posts() ) {
+if( have_posts() ) { 
+?>
+    <div class="masonry-container">
```

Comment:
why use masonry- prefexed classes if the masonry is only used in one context? more semantic to call it the journal container and the post journal posts. or at least have the css hooked to a differerent class